taxonomy/migrations_utils.py

```diff
@@ -0,0 +1,70 @@
+"""
+Utils used in migrations.
+"""
+from django.db.models import Count
+
+
+def remove_all_jobs(apps, schema_editor):  # pylint: disable=unused-argument
+    """
+    Delete all Jobs object as we are adding a unique non-nullable `external_id`, that we cannot back-populate.
+    """
+    job_model = apps.get_model('taxonomy', 'Job')
+    job_model.objects.all().delete()
+
+
+def remove_duplicates_skills(apps, schema_editor):  # pylint: disable=unused-argument
+    """
+    Remove all duplicate records from the Skill table.
+    """
+    skill_table = apps.get_model('taxonomy', 'Skill')
+    duplicate_skills = skill_table.objects.values('external_id') \
+        .annotate(skill_count=Count('id')).order_by() \
+        .filter(skill_count__gt=1)
+    for skill in duplicate_skills:
+        skill_ids = list(skill_table.objects.filter(external_id=skill['external_id']).values_list('id', flat=True))[1:]
```

Comment:
what is the reason for adding `[1:]`?

Reply:
@saleem-latif I am not removing all duplicate records. let's say there are 5 same records, I am leaving the first one, and removing the remaining 4.